### DISCUSSION TITLE
[Question][Hw0] Environment in Mac

### DISCUSSION THREAD

**Discussion by miraliahmadli** [prosyslab-classroom/cs348-information-security#477] [Q&A]
I cannot connect to KCLOUD on my Mac. After login to the kcloud page, it asks me to download vpn profile, but when I click on that it just opens html code of some page. Did anyone face the similar issue? I noticed that kcloud website redirects me to "https://kcloudvpn.kaist.ac.kr/iphone.jsp" which I guess it is for iphone, not mac.

Also, I cannot install the LLVM by using the command given in the slides. The error said it cannot find the Java Runtime. Should I install Java for that? Is it possible if I don't download it at all?

**Comment by miraliahmadli** [comment]
> I cannot connect to KCLOUD on my Mac. After login to the kcloud page, it asks me to download vpn profile, but when I click on that it just opens html code of some page. Did anyone face the similar issue? I noticed that kcloud website redirects me to "https://kcloudvpn.kaist.ac.kr/iphone.jsp" which I guess it is for iphone, not mac.

[Update]
Apparently, installation is slightly different. I followed this guide and it worked: "https://kcloud.kaist.ac.kr/index.php/board/?mod=document&uid=132". But I cannot change the password. Can you help with it?

**Comment by suplookie** [comment]
the script for installing llvm have commands that does not work for Mac, like apt or update-alternatives, if the slide you meant is [https://docs.google.com/presentation/d/1y5Pk-Ks3883ut5WqYRnj8YggxBiAK3ZVzf-hJN6Z8XY/edit#slide=id.gc3a7764c3b_2_4](https://docs.google.com/presentation/d/1y5Pk-Ks3883ut5WqYRnj8YggxBiAK3ZVzf-hJN6Z8XY/edit#slide=id.gc3a7764c3b_2_4)

**Comment by jirheee** [comment]
I downloaded SecuwaySSL U V1.0 as the instruction says, but when I tried kcloudvpn.kaist.ac.kr at the url, it didn't work. What should I type in th url box?

**Comment by miraliahmadli** [comment]
> I downloaded SecuwaySSL U V1.0 as the instruction says, but when I tried kcloudvpn.kaist.ac.kr at the url, it didn't work. What should I type in th url box?

Strange 🤔 Did you try putting https://?  Also, guide uses v2.0 and it works fine.

**Comment by suplookie** [comment]
> I downloaded SecuwaySSL U V1.0 as the instruction says, but when I tried kcloudvpn.kaist.ac.kr at the url, it didn't work. What should I type in th url box?

I used the IP address from the configuration profile I got when I logged in the kcloud. if you log in the kcloud, there will be a download link for your profile

**Comment by 0xXcrypted** [comment]
Struggling with the same issue on kcloud.
I followed the steps mentioned previously and tried using SecuwaySSL U V1. It seems like Secuway SSL fails to connect even using the IP address from the configuration profile. How can I solve this issue.
I'm currently using Macbook15-2018 big sur

**Comment by KihongHeo** [comment]
@junseon0-0 @RiceBiscuits , could you help ?

**Comment by Junseon00** [comment]
We don't use the Mac, so we don't know what the problem is. So, we'll check and let you know ASAP!

**Comment by RiceBiscuits** [comment]
![KakaoTalk_20210312_195204738](https://user-images.githubusercontent.com/11806775/110931844-13659e00-836e-11eb-88f6-6b46b4cdef91.jpg)
Please type 'https://kcloudvpn.kaist.ac.kr' in the url bar instead of your assigned virtual machine IP.
If you have a same problem again, please use version 2.
